documentation/WhitePapers/SustainabilityUseCasesAndLandscape2022.md

```diff
@@ -0,0 +1,252 @@
+
+### Challenges in the Public Cloud
+Public cloud providers, such as AWS, Azure, and GCP are often quite restrictive, as the providers limit decisions users can make in regard to getting sustainability-related metrics, such as the energy consumption, hardware, electricity source, data center PUE, etc.
+Providers try to keep their day-to-day costs/energy usage/emmissions down, but what functionality they expose users to can be quite limited.
+<!--- this statement needs reference: They do not trust their users, as users vary from amateur to experienced. --->  
+They also are not always great at accounting for carbon costs by specific users, and the carbon accounting can take much longer than users have to connect to individual types of jobs.
+The hyperscalers mentioned above offer insight into the carbon emissions of cloud resources through carbon dashboards or APIs.
+Yet, these can be quite limited and/or have a considerable time lag for the carbon emission data to become available.
+In addition, the methodologies used to calculate carbon emissions can vary between cloud providers.
+How this information is measured or estimated is often obtuse.
+
+####  Users of Cloud Service Providers
+Users are often unaware and inexperienced.
+Those that do care about their environmental impact have a hard time connecting their individual workloads to their carbon costs.
+
+### Challenges in the Private Clouds
+These are clouds run by particular companies or universities for use of the members of those companies or universities.
+These clouds are often more trusting environments, as the users are accountable to the administrators or management of the cloud they are running their workloads on.
+
+### Sector specific Challenges
+#### Telco
+Telco customers are often demanding.
+Telco needs their systems to be extremely stable, and the traffic to be fast and reliable.
+Some systems are left entirely at full-power because power modulation can affect traffic and the tolerance for this is low.
+Building systems that reduce power that Telco is confident will not affect their traffic is challenging.
+
+#### Finance
+Finance may have simulations being run, in the off-hours, and those will look like an AI/ML workload.  However,
+for transactions and fast-traffic, finance has predictable times of day of use-when the markets are up.  For this
+reason, time-of-day adjustments on the majority of the clusters run by finance can be limited.  However, transaction
+times do affect real dollars, so being fast will be prioritized in these environments over power use.  In order
+to get this set of customers to use sustainable options, care must be taken to limit the impact to their bottom line.
```

Comment:
Line breaks are off. I prefer one line per sentence to make it easier to comment.